src/libos/src/net/socket/unix/stream/file.rs

```diff
@@ -53,8 +54,18 @@ impl File for Stream {
         }
     }
 
-    fn ioctl(&self, cmd: &mut IoctlCmd) -> Result<i32> {
+    fn ioctl(&self, cmd: &mut IoctlCmd, entry: &mut FileTableEntry) -> Result<i32> {
         match cmd {
+            IoctlCmd::TCGETS(_) => return_errno!(ENOTTY, "not tty device"),
```

Comment:
Could you add FIONBIO for Unix domain socket together?
See https://github.com/liqinggd/occlum/commit/1e0029a95d399aec5b0b23a4bf6774bba94cb06b for the commit.

Reply:
Done